README.md

```diff
@@ -1,2 +1,98 @@
 # v2r
-Vector to Raster interpolation routines 
+Vector to Raster Interpolation Routines 
+
+# Features
+v2r contains two algorithms with an associated testing suite.
+
+- Inverse Distance Weighting (IDW)
+- Cleaner 
+## IDW
+![idw-sample](images/idw_sample.png)
+<br>
+The interpolation becomes more drastic as _p_ increases.
+
+### **Equation** <br>
+$$z_p= \frac{\displaystyle\sum_{i=1}^{n} (\frac {z_i}{d_i^p}) } {\displaystyle\sum_{i=1}^{n} (\frac {1}{d_i^p})}$$
+<br>
+
+**Read from Geopackage** <br>
+`./main idw -g -f [FILEPATH] --layer [LAYERNAME] --field [FIELDNAME]`
```

Comment:
Per the prior comment, please update all refs to the binary (e.g. this should read  `./v2r idw -g -f [FILEPATH] --layer [LAYERNAME] --field [FIELDNAME]`